packages/cli/lang/en.lyaml

```diff
@@ -517,6 +517,7 @@ en:
               deploy: "Deploy error: {{ details }}"
               noBuilds: "Deploy error: no builds for this project were found."
               noBuildId: "You must specify a build to deploy"
+              projectNotFound: "{{ projectName }} does not exist in account {{ accountId }}. Run `{{#yellow}}hs project upload{{/yellow}}` to upload your project files to HubSpot."
```

Comment:
There's also a UI util for logging command references: [uiCommandReference](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/ui.js#L97)